### PR TITLE
Fix/1185 mapping revisions

### LINKED ISSUE #1185: UI: mapping revisions
## Expected Behavior
* When a computation has no fields to map the maps list page should not show the edit button, which just leads to a blank page
* console warnings when entering the maps page, might need a computation with mappable fields (ie ssr) to see

### PR DESCRIPTION
**Pull request by kalitopix**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our [guidelines](../CONTRIBUTING.md)
- [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
- [x] Passes lint

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Bug fix

* **What is the current behavior?** (You can also link to an open issue here)
closes #1185

* **What is the new behavior (if this is a feature change)?**
Updated mapping UI

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Other information**:

### CODE REVIEW

File: packages/coinstac-graphql-schema/src/queries.js [hunk 1]
```diff
@@ -228,21 +228,6 @@ const queries = {
       }
     }
   `,
-  fetchRunStatus: `
```
Comment (rssk) [inline packages/coinstac-graphql-schema/src/queries.js]: Why remove these?

**Review by rssk — CHANGES_REQUESTED** [review]

File: packages/coinstac-graphql-schema/src/queries.js (hunk 1 above)
Comment (kalitopix) [reply packages/coinstac-graphql-schema/src/queries.js]: @rssk these two are duplicated.

Screenshot 2022-02-16 at 12 00 50 PM

Comment (rssk) [reply packages/coinstac-graphql-schema/src/queries.js]: ah cool 👍

**Review by rssk — CHANGES_REQUESTED** [review]
needs to have the edit button visible on computations that have mappable fields that have already been mapped so they can be viewed etc

**Comment by rssk** [comment]
outdated

**Pull request closed without merging**